### PR TITLE
Add codacy badge wrt #146

### PR DESCRIPTION
**Pull request by scarabeusiv**

### CODE REVIEW

**Comment by scarabeusiv** [comment]
@dirkmueller it is pointing to my codacy namespace but testing the rpmlint from the main project.

See commits here:
https://app.codacy.com/app/scarabeusiv/rpmlint/commits?bid=8405208

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # rpmlint
 
 [![Build Status](https://travis-ci.org/rpm-software-management/rpmlint.svg)](https://travis-ci.org/rpm-software-management/rpmlint)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/3539541ffbaf442ab34a098d30af166f)](https://www.codacy.com/app/scarabeusiv/rpmlint?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=rpm-software-management/rpmlint&amp;utm_campaign=Badge_Grade)
```
Comment (Conan-Kudo) [inline README.md]: Is it not possible to enable the app under the organization rather than from you?

Comment (Conan-Kudo) [reply README.md]: @dmach Is this something we can do?

**Comment by scarabeusiv** [comment]
Crap I am doing my WIP stuff in wrong branch...

**Comment by scarabeusiv** [comment]
As it is 1 year without activity I am going to abandon the codacy, since it is just nice to have no need to leave it hanging.

**Pull request closed without merging**